### PR TITLE
Post launch sw

### PR DESCRIPTION
**Pull request by tanyaparker**
Teacher dashboard:

image

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/starwars.haml [hunk 1]
```diff
@@ -43,8 +43,8 @@ social:
       %div.sw-tutorial-info
         %p.sw-tutorial-description= I18n.t(:starwars_blocks_description)
         %p.sw-tutorial-specs= I18n.t(:starwars_blocks_specs)
-      %a{:href => '#'}
-        %button.disabled.sw-disabled{disabled:"disabled"}= I18n.t(:starwars_blocks_start_button)
+      %a{:href => CDO.studio_url('/s/starwarsblocks/reset'), :target=>'_self'}
+        %button.sw-try-button= I18n.t(:starwars_blocks_start_button)
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/starwars.haml:47]: I updated the string in the i18n gsheet

Comment (caleybrock) [inline pegasus/sites.v3/code.org/public/starwars.haml:47]: now it's updated on staging, but disabled, is that okay?

Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/starwars.haml:47]: Well if this is OK to merge in asap, it won't matter.

File: dashboard/app/views/api/hoc_courses.html.haml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 .row{style: 'margin: 0px;'}
+  = render partial: 'shared/course_tall_block', locals: { id: Script::STARWARS_NAME, showState: false }
   = render partial: 'shared/course_tall_block', locals: { id: Script::HOC_NAME, showState: false }
   = render partial: 'shared/course_tall_block', locals: { id: Script::FROZEN_NAME, showState: false }
-  = render partial: 'shared/course_tall_block', locals: { id: Script::INFINITY_NAME, showState: false }
```
Comment (caleybrock) [inline dashboard/app/views/api/hoc_courses.html.haml:4]: is this used elsewhere? are we removing it everywhere?

Comment (tanyaparker) [inline dashboard/app/views/api/hoc_courses.html.haml:4]: Nope, this is the view used only on the teacher dashboard.

**Comment by caleybrock** [comment]
I'd say this lgtm but maybe @breville should take a quick look too

File: dashboard/config/locales/data.en.yml [hunk 3]
```diff
@@ -509,10 +509,13 @@ en:
         'hoc2015_preview_blocks' : 'Make your own game blocks'
         'hoc2015_preview_typing' : 'Make your own game typing'
         'hoc2015_preview_conclusion' : 'Make your own game conclusion'
-        'starwars_intro' : 'Introduction'
-        'starwars_typing' : 'JavaScript'
-        'starwars_events' : 'Events'
-        'starwars_congrats' : 'Congratulations'
+        'starwars_intro' : 'Star Wars Hour of Code - Introduction'
```
Comment (breville) [inline dashboard/config/locales/data.en.yml]: The official name is Star Wars: Building a Galaxy With Code.. for what it's worth.

**Comment by breville** [comment]
I didn't read the entire transcript, but generally things lgtm.

File: dashboard/app/views/shared/_course_tall_block.haml [hunk 4]
```diff
@@ -31,6 +31,7 @@
     - title = t('upsell.frozen.title')
   - elsif id == 'starwars' || id == 'starwarsblocks'
     - title = t('upsell.starwars.title')
+    - url = 'https://code.org/starwars'
```
Comment (breville) [inline dashboard/app/views/shared/_course_tall_block.haml:34]: can we use a CDO function to make this work in various envs?

**Pull request merged**